src/lsp/cobol_ptree/branching_statements.ml

```diff
@@ -31,6 +31,16 @@ type goto_stmt =
 let pp_goto_stmt ppf { goto_target } =
   Fmt.pf ppf "@[GO TO @[%a@]" pp_procedure_name' goto_target
 
+(* GOTO ENTRY *)
+type goto_entry =
```

Comment:
One last thing: `goto_entry_stmt` instead of just `goto_entry` (and appropriately renamed in visitors).

Reply:
Actually, I didn't pay enough attention before, but I don't see any point in having four distinct constructors for `GoTo*`. At worse we should have `GoTo` and `GoToEntry`, but the `DEPENDING` part is not really relevant.
The distinctions should come in `goto_stmt`, with dedicated constructors and inline records.